### PR TITLE
Fix TypeScript compat issue

### PR DESCRIPTION
**Pull request by jasonmalinowski**
<details><summary>Ask Mode template</summary>

### Customer scenario

Try using Quick Info in a TypeScript file. VS crashes.

### Bugs this fixes

(but not filed)

### Workarounds, if any

None.

### Risk

Low, just adding a constructor back.

### Performance impact

None.

### Is this a regression from a previous update?

Yes.

### Root cause analysis

This was removed, and caught during validation builds. We have validation in validation builds to catch these errors, but it can't be caught sooner.

### How was the bug found?

Automated testing.

</details>

### CODE REVIEW

File: src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs [hunk 1]
```diff
@@ -897,6 +897,11 @@ private void OnSuggestedActionsChanged(Workspace currentWorkspace, DocumentId cu
                 using (var asyncToken = _owner.OperationListener.BeginAsyncOperation(nameof(GetSuggestedActionCategoriesAsync)))
                 {
                     var document = range.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
+                    if (document == null)
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs]: This change was merged as a part of #24937 but I had it in my PR too to validate locally.

**Comment by jasonmalinowski** [comment]
Tagging @jinujoseph / @Pilchie for review and ask mode signoff.

File: src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/DeferredContent/QuickInfoDisplayDeferredContent.cs [hunk 2]
```diff
@@ -16,6 +15,29 @@ internal class QuickInfoDisplayDeferredContent : IDeferredQuickInfoContent
         public IDeferredQuickInfoContent ExceptionText { get; }
         public IDeferredQuickInfoContent WarningGlyph { get; }
 
+        // DO NOT REMOVE: compat for Typescript
+        public QuickInfoDisplayDeferredContent(
```
Comment (Therzok) [inline src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/DeferredContent/QuickInfoDisplayDeferredContent.cs]: Isn't this constructor and the one below essentially the same parameter wise?

Comment (jasonmalinowski) [reply src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/DeferredContent/QuickInfoDisplayDeferredContent.cs]: There's one less parameter. :-)

**Pull request merged**